miqa/learning/nn_classifier.py

```diff
@@ -302,6 +301,47 @@ def evaluate1(model, image_path):
     return labeled_results
 
 
+def evaluateMany(model, image_paths):
+    itk_reader = monai.data.ITKReader()
+    # Define transforms for image
+    train_transforms = Compose(
+        [
+            LoadImaged(keys=['img'], reader=itk_reader),
+            EnsureChannelFirstd(keys=['img']),
+            ScaleIntensityd(keys=['img']),
+            ToTensord(keys=['img']),
+        ]
+    )
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+    results = {}
+    for image_path in image_paths:
+        evaluation_ds = monai.data.Dataset(
+            data=[
+                {
+                    'img': image_path,
+                    'info': torch.FloatTensor([0] * (regression_count + len(artifacts))),
+                }
+            ],
+            transform=train_transforms,
+        )
+        evaluation_loader = DataLoader(
+            evaluation_ds, batch_size=1, pin_memory=torch.cuda.is_available()
+        )
+
+        tensor_output = evaluate_model(model, evaluation_loader, device, None, 0, 'evaluate1')
+        result = tensor_output.cpu().tolist()[0]
```

Comment:
Once you have the results, they should be for the entire batch (all the files in `image_paths`).